kernel_gateway/gatewayapp.py

```diff
@@ -221,6 +211,21 @@ def _api_changed(self, name, old, new):
     def allow_notebook_download_default(self):
         return os.getenv(self.allow_notebook_download_env, 'False') == 'True'
 
+    def _load_api_module(self, module_name):
+        '''Tries to import the given module name'''
+        api_module = None
+        # some compatibility allowances
+        if module_name == 'jupyter-websocket':
+            module_name = 'kernel_gateway.jupyter_websocket'
+        elif module_name == 'notebook-http':
+            module_name = 'kernel_gateway.notebook_http'
+        try:
+            api_module = importlib.import_module(module_name)
+        except ImportError:
+            # a ValueError will be triggered in the caller
+            pass
```

Comment:
Better to let the ImportError bubble. It's possible the user has provided a valid package name, but that package cannot be imported for other reasons (e.g., it is throwing an exception because of a missing dependency).